pageTests/admin/archive-a-ward-confirmation.test.js

```diff
@@ -0,0 +1,32 @@
+import { getServerSideProps } from "../../pages/admin/archive-a-ward-confirmation";
+
+// TODO: This needs to be moved once the verifyToken logic is in the container..
```

Comment:
Is the verifyToken logic now in the container?

Reply:
See other resolved conversation.